root/layout/components/Footer.js

```diff
@@ -9,87 +9,568 @@
+        </a>
+      </li>
+      <li className="nav-item dropdown fs-4">
+        <a href="https://www.transifex.com/musicbrainz/musicbrainz/">
+          {l('Help Translate')}
+        </a>
+      </li>
+    </ul>
+  </div>
+);
+
+const Footer = (): React.Element<'section'> => {
   const $c = React.useContext(CatalystContext);
-  const stash = $c.stash;
+  const serverLanguages = $c.stash.server_languages;
+
   return (
-    <div id="footer">
-      <p className="left" id="footer-menu">
-        <a className="internal" href={DONATE_URL}>{l('Donate')}</a>
-        <a className="internal" href="//wiki.musicbrainz.org/">{l('Wiki')}</a>
-        <a className="internal" href="https://community.metabrainz.org/">{l('Forums')}</a>
-        <a className="internal" href="/doc/Communication/IRC">
-          {l('Chat (IRC)')}
-        </a>
-        <a className="internal" href="http://tickets.metabrainz.org/">{l('Bug Tracker')}</a>
-        <a className="internal" href="https://blog.metabrainz.org/">{l('Blog')}</a>
-        <a className="internal" href="https://twitter.com/MusicBrainz">{l('Twitter')}</a>
-
-        {DBDefs.BETA_REDIRECT_HOSTNAME ? (
-          <a
-            className="internal"
-            href={
-              '/set-beta-preference?' + returnToCurrentPage($c)
-            }
-          >
-            {DBDefs.IS_BETA
-              ? l('Stop using beta site')
-              : l('Use beta site')}
-          </a>
-        ) : null}
-      </p>
-
-      <p className="right">
-        {exp.l(
-          `Brought to you by {MeB|MetaBrainz Foundation} and our
-           {spon|sponsors} and {supp|supporters}. Cover Art provided
-           by the {caa|Cover Art Archive}.`,
-          {
-            caa: '//coverartarchive.org/',
-            MeB: 'https://metabrainz.org/',
-            spon: 'https://metabrainz.org/sponsors',
-            supp: 'https://metabrainz.org/supporters',
-          },
-        )}
-
-        {DBDefs.DB_STAGING_SERVER && DBDefs.GIT_BRANCH ? (
-          <>
+    <section className="footer">
+      <div className="ms-4 me-4 mb-3 ps-4 pb-3 pe-4 border-top container">
+        <div className="row mb-4 mt-2 pt-2">
+          <div className="col-sm-12 col-md-4">
+            <h3>
+              <img
+                alt="MusicBrainz"
+                src="../../static/images/meb-logos/musicbrainz.svg"
+                width="180"
+              />
+            </h3>
             <br />
-            {exp.l('Running: {git_details}', {
-              git_details: (
-                <span
-                  className="tooltip"
-                  key="git_details"
-                  title={DBDefs.GIT_MSG}
-                >
-                  {DBDefs.GIT_BRANCH}
-                  {' '}
-                  {bracketedText(DBDefs.GIT_SHA)}
+            <p className="fs-4">
+              {l(`MusicBrainz is an open music encyclopedia that collects
+              music metadata and makes it available to the public.`)}
```

Comment:
I'm not sure if we need an "about" blurb in the footer, because anyone reading this already knows what MusicBrainz is. @reosarevok @yvanzo thoughts?

Reply:
Removed this and updated the UI! Check the updated screenshots in the PR :)